# Algortihm Parameters
alpha: 0.1
gamma: 0.8 # Higher gamma = smaller discount means care more about long term reward
           # Small gamma = High discount means care more about short term reward.
epsilon_start: 1.0
epsilon_stop: 0.01
decay_rate: 0.01
decay_step: 0.0
nepisodes: 2000
nsteps: 32
running_step: 0.5   # in seconds

# Environment Parameters
desired_pose:
    x: 8.0
    y: 1.0
    z: 0.7
max_altitude: 2.0   # in meters
max_incl: 0.7       # in rads
speed_value: 1.0    # in m/s
